src/logic/calculate.js

```diff
@@ -0,0 +1,22 @@
+import operate from './operate';
+
+const calculate = (
+  data = {
+    total: '',
+    next: '',
+    operation: '',
+  },
+  buttonName,
+) => {
+  let { total, next } = data;
+
+  if (buttonName === '+/-') {
+    total *= -1;
+    next *= -1;
+    return next;
+  }
```

Comment:
- [ ] Please add conditionals for other button names. Like numbers and decimal points.